code/modules/spells/spell_types/wizard.dm

```diff
@@ -261,17 +261,65 @@
 	invocation_type = "shout"
 	range = 20
 	cooldown_min = 20 //10 deciseconds reduction per rank
-
+	var/fireball_type = /obj/item/projectile/magic/fireball

```

Comment:
I fixed that now, it will seek out next tile projectiles as expected, making this an even **more dangerous spell!** I can smell the burning salt now!